packages/editor/package.json

```diff
@@ -40,6 +40,7 @@
 		"@wordpress/data-controls": "file:../data-controls",
 		"@wordpress/date": "file:../date",
 		"@wordpress/deprecated": "file:../deprecated",
+		"@wordpress/dom": "file:../dom",
```

Comment:
Is this needed? `editor` already depends on `block-editor`, which is the provider of the hook.